api/v1/announcements/PKPAnnouncementHandler.inc.php

```diff
@@ -127,56 +122,38 @@ public function get($slimRequest, $response, $args)
      */
     public function getMany($slimRequest, $response, $args)
     {
-        $request = Application::get()->getRequest();
-
-        $params = [
-            'count' => 30,
-            'offset' => 0,
-        ];
+        $collector = Query::announcement()->getCollector()
+            ->limit(30)
```

Comment:
Could be a constant or env configurable value?

Reply:
I think we want to be able to specify different defaults for different endpoints. But it could be a constant in the handler. I'll move it there, thanks.